zebrad/tests/acceptance.rs

```diff
@@ -2261,7 +2274,7 @@ fn zebra_rpc_conflict() -> Result<()> {
     // But they will have different Zcash listeners (auto port) and states (ephemeral)
     let dir2 = testdir()?.with_config(&mut config)?;
 
-    check_config_conflict(dir1, regex1.as_str(), dir2, "Unable to start RPC server")?;
+    check_config_conflict(dir1, regex1.as_str(), dir2, "Address already in use")?;
```

Comment:
We should update `ZEBRA_FAILURE_MESSAGES`/`ZEBRA_CHECKPOINTS_FAILURE_MESSAGES` too.

Reply:
Thanks for noticing this.

Reply:
Addressed in [Update error log messages](https://github.com/ZcashFoundation/zebra/pull/9459/commits/113d1aa0408b3fb979eda288c16122c9fbb8a3d3).